lib/v2/hnu/radar.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'hnu.edu.cn': {
+        _name: '湖南大学',
+        scc: [
+            {
+                title: '湖南大学 校内招聘',
```

Comment:
```suggestion
                title: '校内招聘',
```

Reply:
I'm sorry, I just understood what you meant. This was my mistake due to carelessness. Link should not be an API interface, and thank you very much for pointing out this bug. It has been fixed. Thank you.